packages/kbn-test/src/jest/setup/emotion.js

```diff
@@ -12,3 +12,16 @@ module.exports = createSerializer({
   classNameReplacer: (className) => className,
   includeStyles: false,
 });
+
+console.error = (message) => {
+  // @see https://github.com/emotion-js/emotion/issues/1105
```

Comment:
This looks like it's going to disable all console.error logs

example from [this PR](https://buildkite.com/elastic/kibana-pull-request/builds/50888#018163e2-9e0b-4b5c-a35e-28f83ce772a1/1928-1936) vs [main](https://buildkite.com/elastic/kibana-on-merge/builds/17284#018163ce-40fa-410a-8977-07e0c6bc9a10/7267-7275)

Reply:
Good catch; will update.

Reply:
Updated in [7363c69](https://github.com/elastic/kibana/pull/133927/commits/7363c697a400306bfd75b37f07b84a500a53b748)